src/macros.js

```diff
@@ -845,6 +845,12 @@ defineMacro("\\rBrace", "\\html@mathml{" +
 defineMacro("\u2983", "\\lBrace"); // blackboard bold {
 defineMacro("\u2984", "\\rBrace"); // blackboard bold }
 
+// The stmaryrd function `\minuso` provides a "Plimsoll" symbol that
+// superimposes the characters \circ and mathminus. Used in chemistry.
+defineMacro("\\minuso", `\\mathbin{\\html@mathml{{\\mathrlap{\\mathchoice{
+    \\kern{0.145em}}{\\kern{0.145em}}{\\kern{0.1015em}}{\\kern{0.0725em}}
+    \\circ}{-}}}{\u29B5}}`);
```

Comment:
Instead of `\u29B5`, we should probably write `\char`⦵` to be consistent with other macros defined in this file.

Also, you're currently not mapping Unicode input of ⦵ to this character. Perhaps you want to add `defineMacro("⦵", "\\minuso")`?  (This will only work with the added `\char` escape above.)

Reply:
I'll make the \char⦵ change.

The mapping of the Unicode code point takes place in `symbols.js`, where I have set the `acceptUnicodeChar` to `true`.

Reply:
Oops, strike that last comment. I had this confused with a different PR. 

Yes, I should map that Unicode value. On it.

Reply:
There are now two copies of `defineMacro("\\minuso"`.  Could you delete the old one?

Reply:
...slaps forehead...

Done.